src/components/Internships/InternshipHero.astro

```diff
@@ -0,0 +1,27 @@
+---
+import CountUpComponent from "../Hero/CountUpComponent";
+---
+
+<div class="flex lg:flex-row flex-col items-center gap-12 w-full mt-12 mb-12">
+  <div class="w-full lg:order-1 order-2">
+    <div class="mb-6 rounded-xl">
+      <img src="/InternshipHero/img_1.png" class="rounded-xl" alt="image-of-a-female-student-pointing-towards-a-laptop-screen" />
+    </div>
+    <div class="flex sm:flex-row flex-col items-center w-full gap-3">
```

Comment:
The two count boxes are not aligning with the image ty to fix that

Reply:
Fixed now